src/app/action/action.html

```diff
@@ -1,5 +1,5 @@
 
-<h2>Action Items!</h2>
+<h2 class="group-header">Action Items!</h2>
```

Comment:
Initially I was doing the styling for action part and it was supposed to look like board, that's why I used the same styling.. but now Abhimanyu is handling the action html and I have asked him to use different classes and different styling.